scout/models/omics_variant.py

```diff
@@ -9,7 +9,7 @@
 from datetime import datetime
 from typing import List, Optional
 
-from pydantic import BaseModel, Field, field_validator, model_validator
+from pydantic import AliasChoices, BaseModel, Field, field_validator, model_validator
```

Comment:
👀

Reply:
Trying to make it accept two different ones to make it backwards compatible. Apparently the variable name does not become an alternative in validation if there is an alias...

Reply:
Ok, now parses and loads both the current demo and older version tomte files (from June, with `hgnc_id` instead of `hgncId` etc).
![Screenshot 2024-09-03 at 13 44 32](https://github.com/user-attachments/assets/a3b1f304-7bbe-49a4-a44f-195b2624286d)
![Screenshot 2024-09-03 at 13 44 19](https://github.com/user-attachments/assets/83e15b45-3285-4c7c-926e-a400a6e8a472)
![Screenshot 2024-09-03 at 13 45 38](https://github.com/user-attachments/assets/4e09cb3a-cbd8-4cb9-be7f-05d201a3a553)
![Screenshot 2024-09-03 at 13 46 51](https://github.com/user-attachments/assets/cd2f1464-e094-484c-b1a2-60622341364f)

Reply:
Nice! I checked the code downstream and looks like sampleID was not used, for instance in the omics variantS template we use case individuals -> https://github.com/Clinical-Genomics/scout/blob/34944ee153c33e14a57931f044f6f4bdf6ec74e8/scout/server/blueprints/omics_variants/templates/omics_variants/outliers.html#L112

I wonder if this is correct .. 🤔 like what if it's not the the first individual of the case to have the variant? Why did we reason in this way?

Reply:
Hm, it is used alright prior to that, in connecting the sample to case individual e.g. https://github.com/Clinical-Genomics/scout/blob/34944ee153c33e14a57931f044f6f4bdf6ec74e8/scout/adapter/mongo/omics_variant.py#L42. Maybe an old dev exception that never got updated to prod. We'll fix! 😁